js/web/script/parse-profiler.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+
+// parse-profiler
+//
+// this script is used to parse performance profiling result.
+// usage:
+// STEP.1 - profiling
+// > npm test -- model test/test-data/{path-to-my-model} --backend={cpu/webgl/wasm} --profile > profile.raw.log
```

Comment:
Personally, i feel it would be better to have the steps integrated with --profile option, so instead of manually redirecting the output stream to a file and running the tool to parse it, it will just generate a parsed file called profile.log or something.

Reply:
agree. but maybe we can have another iteration of improvement, and keep it as is for now in order to make the 1.8 release.

Reply:
sounds good.